### ISSUE TITLE
Travelogue

### ISSUE THREAD

**Issue by muffinista** [dariusk/NaNoGenMo-2015#13]
I'm in, my plan is to generate a travel journal of some sort.

**Comment by bcj** [comment]
That is an awesome idea

**Comment by dariusk** [comment]
Hi, I'm going through and updating the titles on issues to make them more specific. Feel free to edit my edit if it's not to your liking. This is to make browsing issues a lot more pleasant.

**Comment by cpressey** [comment]
I remember thinking when I read Kerouac's _On the road_ last year that there were fairly long stretches in it that seemed quite amenable to generation.

"I got off the truck in Foosberg and I had 80 cents in my pocket.  I spent 5 cents on coffee and 15 cents on a slice of pie and fell asleep in a field.  The next day, I hitched a ride with a fellow in a blue Buick.  In the back was the beatest dog I ever laid eyes on.  He could only take me as far as Barsville..."

**Comment by muffinista** [comment]
Yeah I actually tried that in 2013 but ran out of time I think, and I know there's at least one generative-style OTR out in the wild. I think this will be a little different, I'll try and add details here before I actually write any code hehe.
